### PR TITLE
Remove reference to localhost

### LINKED ISSUE #93: Link in production site points to localhost
Some of the internal links are pointing to localhost. In this case, we have the 1st testing blog post pointing to a local link to the second.

### PR DESCRIPTION
**Pull request by taras**
Fixes #93

### CODE REVIEW

**Review by cowboyd — COMMENTED** [review]
Why not use a relative link so that it works regardless of environment?

File: src/pages/blog/2020-triple-threat-to-testing-part-1-speed.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ We think it’s worth the effort to improve testing, as speedy test suites short
 
 Speed is critical to a healthy testing strategy, but it won't work without the other pillars of the testing triple threat. Next we focus on the second essential make-or-break factor in testing: reliability. Tests can be flaky, but there are strategies you can take now to detect, fix, and even prevent having non-deterministic tests in your application.
 
-Keep on reading, [The Triple Threat to Testing — Part 2: Reliability](http://localhost:8000/blog/2020-triple-threat-to-testing-part-2-reliability/).
+Keep on reading, [The Triple Threat to Testing — Part 2: Reliability](http://frontside.com/blog/2020-triple-threat-to-testing-part-2-reliability/).
```
Comment (cowboyd) [inline src/pages/blog/2020-triple-threat-to-testing-part-1-speed.md:143]: I don't know if relative paths work in Gatsby, but 🤷 
```suggestion
Keep on reading, [The Triple Threat to Testing — Part 2: Reliability](2020-triple-threat-to-testing-part-2-reliability/).
```

**Comment by jorgelainfiesta** [comment]
You need to add the trailing slash it seems. I

**Review by jorgelainfiesta — APPROVED** [review]
tak for dette!

File: src/pages/blog/2020-triple-threat-to-testing-part-1-speed.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ We think it’s worth the effort to improve testing, as speedy test suites short
 
 Speed is critical to a healthy testing strategy, but it won't work without the other pillars of the testing triple threat. Next we focus on the second essential make-or-break factor in testing: reliability. Tests can be flaky, but there are strategies you can take now to detect, fix, and even prevent having non-deterministic tests in your application.
 
-Keep on reading, [The Triple Threat to Testing — Part 2: Reliability](http://localhost:8000/blog/2020-triple-threat-to-testing-part-2-reliability/).
+Keep on reading, [The Triple Threat to Testing — Part 2: Reliability](/blog/2020-triple-threat-to-testing-part-2-reliability/).
```
Comment (cowboyd) [inline src/pages/blog/2020-triple-threat-to-testing-part-1-speed.md:143]: this is relative to the host, but not to the path, so it won't work if we want to mound this site under a path at a different place.... shouldn't be a problem, but suprised that the fully relative didn't work. I wonder if './2020/triple-threat' would work.

**Pull request merged**